ldi-core/version-object-creator/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/VersionObjectCreator.java

```diff
@@ -47,8 +47,11 @@ private MemberInfo extractMemberInfo(Model linkedDataModel, Resource memberTypeR
 			Property dateObservedProperty) {
 		String dateObserved = Optional.ofNullable(dateObservedProperty)
 				.map(property -> linkedDataModel.listStatements(null, property, (Resource) null)
+						.mapWith(Statement::getObject)
+						.filterKeep(RDFNode::isLiteral)
+						.mapWith(RDFNode::asLiteral)
+						.filterKeep(literal -> literal.getDatatype().getURI().contains("DateTime"))
```

Comment:
Will this always be "DateTime"? Considering we provide a dateObservedProperty?

Reply:
the datatype (^^datatype) should somehow still be a datetime of some sort. I already made it a bit more lenient for not only supporting xsd:Datetime for the ngsi-ld case, but since this is going to be a future generatedAtTime property, this should be enforced i believe